tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js

```diff
@@ -34,81 +42,100 @@ describe('aws-node-sdk v2auth query tests', function testSuite() {
 
     before(() => {
         const config = getConfig('default', { signatureVersion: 'v2' });
-
-        s3 = new S3(config);
+        s3 = new S3Client(config);
     });
 
     // AWS allows an expiry further in the future
     // 604810 seconds is higher that the Expires time limit: 604800 seconds
     // ( seven days)
     itSkipAWS('should return an error code if expires header is too far ' +
-        'in the future', done => {
-        const params = { Bucket: bucket, Expires: 604810 };
-        const url = s3.getSignedUrl('createBucket', params);
-        provideRawOutput(['-verbose', '-X', 'PUT', url], httpCode => {
-            assert.strictEqual(httpCode, '403 FORBIDDEN');
-            done();
+        'in the future', async () => {
+        
+        // First, get a valid signed URL with maximum allowed expiry
```

Comment:
```suggestion
        // First, get a valid signed URL with maximum allowed expiry of 7 days
```

then remove the other comment, or create a variable named 7daysInSec or something

Also the comment line 60 about the "10 seconds more than limit is weird", because + 604810 means + 7days, not +10 seconds 🤔

Reply:
ah nvm its 
604810
not
604800
hard to spot, whatever, its nit you can leave it